traversability_estimation:
  ros__parameters:
    submap_service: "/get_raw_submap" #"/elevation_mapping_long_range/get_submap" #"/elevation_mapping/get_grid_map" #
    map_frame_id: "map"  # Must be the same as the input elevation submap.
    robot_frame_id: "base_link"
    min_update_rate: 5.0 #0.2
    map_center_x: 0.0
    map_center_y: 0.0
    map_length_x: 10.0
    map_length_y: 10.0
    footprint_yaw: 0.7854
    max_gap_width: 0.2
    grid_map_to_initialize_traversability_map:
      enable: true
      grid_map_topic_name: elevation_map